tools/pkg/engine_build_configs/bin/run.dart

```diff
@@ -0,0 +1,141 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io' as io;
+
+import 'package:engine_build_configs/engine_build_configs.dart';
+import 'package:engine_repo_tools/engine_repo_tools.dart';
+import 'package:path/path.dart' as p;
+import 'package:platform/platform.dart';
+import 'package:process_runner/process_runner.dart';
+
+// This is an example of how to use the APIs of this library to parse and
+// execute the build configurations json files under ci/builders.
+//
+// Usage:
+// $ dart bin/run.dart [build config name] [build name]
+// For example:
+// $ dart bin/run.dart mac_unopt host_debug_unopt
+//
+// The build config names are the namess of the json files under ci/builders
+// The build names are the "name" fields of the maps in the list of "builds".
+
+void main(List<String> args) async {
+  final String? configName;
+  final String? buildName;
+  if (args.length >= 2) {
+    configName = args[0];
+    buildName = args[1];
+  } else {
+    io.stderr.writeln('Wrong arguments');
+    io.exitCode = 1;
+    return;
+  }
+
+  // Find the engine repo.
+  final Engine engine;
+  try {
+    engine = Engine.findWithin();
+  } catch (e) {
+    io.stderr.writeln(e);
+    io.exitCode = 1;
+    return;
+  }
+
+  // Find and parse the engine build configs.
+  final io.Directory buildConfigsDir = io.Directory(p.join(
+    engine.flutterDir.path, 'ci', 'builders',
+  ));
+  final BuildConfigLoader loader = BuildConfigLoader(
+    buildConfigsDir: buildConfigsDir,
+  );
+
+  // Treat it as an error if no build configs were found. The caller likely
+  // expected to find some.
+  final Map<String, BuildConfig> configs = loader.configs;
+  if (configs.isEmpty) {
+    io.stderr.writeln(
+      'Error: No build configs found under ${buildConfigsDir.path}',
+    );
+    io.exitCode = 1;
+    return;
+  }
+  if (loader.errors.isNotEmpty) {
+    loader.errors.forEach(io.stderr.writeln);
+    io.exitCode = 1;
+  }
+
+  // Check the parsed build configs for validity.
+  BuildConfig? targetConfig;
```

Comment:
replace the loop below with:

targetConfig = configs[name];

Reply:
Thanks. Done.